src/pages/components/tipSection/index.js

```diff
@@ -1,11 +1,40 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import Tips from './Tips';
 import IdeaIcon from '../icons/IdeaIcon';
+import Demo from './Demo';
 
 const TipSection = () => {
   const { t } = useTranslation('features');
 
+  const [isDemo, setIsDemo] = useState(false);
+  const [isActive, setIsActive] = useState(true);
+  const [isNotActive, setIsNotActive] = useState(true);
+
+  useEffect(() => {
+    if (isDemo === false) {
+      setIsActive(
+        'h-[38px] w-20 rounded-l-lg bg-navy-600 px-2 py-1 text-lg font-semibold text-white hover:underline focus:bg-navy-600 focus:text-white',
+      );
+    } else {
+      setIsActive(
+        'h-[38px] w-20 rounded-l-lg bg-navy-100 px-2 py-1 text-lg font-semibold text-dark-ash-900 hover:underline focus:bg-navy-600 focus:text-white',
+      );
+    }
+  }, [isDemo]);
+
+  useEffect(() => {
+    if (isDemo === true) {
```

Comment:
```suggestion
    if (isDemo) {
```
@CaneteJoash this one too